GTViewController/Classes/Custom Classes/GTFollowupThankYouView.m

```diff
@@ -0,0 +1,60 @@
+//
+//  GTFollowupThankYouView.m
+//  GTViewController
+//
+//  Created by Ryan Carlson on 4/12/16.
+//  Copyright © 2016 Michael Harrison. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "GTFollowupThankYouView.h"
+#import "GTLabel.h"
+
+#import "GTPageInterpreter.h"
+#import "UISnuffleButton.h"
+
+@implementation GTFollowupThankYouView
+
+- (instancetype) initFromElement:(TBXMLElement *) thankYouComponentElement withFrame:(CGRect)frame withPageStyle:(GTPageStyle *)style {
+    
+    self = [super initWithFrame:frame];
+    [self setBackgroundColor:style.backgroundColor];
+    
+    int thankYouCurrentY = 20;
+    
+    TBXMLElement *thankYouElement = thankYouComponentElement->firstChild;
+    
+    while (thankYouElement) {
```

Comment:
this method is getting big. you might want to consider moving this logic to its own method